Source/SwiftyDropbox/Platform/SwiftyDropbox_iOS/OAuthMobile.swift

```diff
@@ -146,6 +196,24 @@ open class DropboxMobileOAuthManager: DropboxOAuthManager {
         }
         return false
     }
+
+    /// Creates a string that contains all code flow query parameters.
+    private static func createExtraQueryParamsString(for authSession: AuthSession) -> String {
```

Comment:
also weird that its static and you reference it by `Self.create..`

Reply:
I think we do this to reference a static method all the time in our internal code base?